src/app/components/Text/__snapshots__/index.test.jsx.snap

```diff
@@ -65,7 +65,7 @@ exports[`Text should render a <figcaption> as per the Caption component override
   <span
     className="c1"
   >
-    Image caption, 
+    
```

Comment:
This feels like it shouldn't of changed if the test correctly was passed the context

Reply:
The test is for the Text _component_ -- the context logic should no longer really exist at this layer. If wanted, I could change it to basically be an exact copy of how the (including duplicating the higher order component CaptionWithOffscreenText), but I wasn't sure that was necessary. Thoughts?

Reply:
If the higher order component was exported could it not be reused by the test (and story if a caption story is needed in text - probably shouldn't have one)? That said, I do get your point that the Text component shouldn't really be aware of the caption.

Reply:
Discussed this in person. We came to the realisation that the Caption container is now testing the paragraph override of Caption anyway, & there is no reason for the Text component to be testing every single variation of it's usage.